draft-ietf-rtcweb-jsep.xml

```diff
@@ -1880,7 +1889,12 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
               the "active" role is RECOMMENDED.</t>
 
               <t>If present in the offer, an "a=rtcp-mux" line, as specified
-              in <xref target="RFC5761"></xref>, Section 5.1.1.</t>
+              in <xref target="RFC5761"></xref>, Section 5.1.1. If the
+              "max-mux" RTCP multiplexing policy is set and no "a=rtcp-mux"
+              line is present in the offer,  then the m=line MUST be
+              marked as rejected by setting the port in the m= line to zero, as
+              indicated in <xref target="RFC3264"></xref>, Section 6.
+              </t>
```

Comment:
This makes sense, as we discussed today, but this raises an interesting point, namely what should the behavior be for an endpoint marked as max-bundle or balanced, when it receives an offer without BUNDLE? If we extend this to its logical conclusion, any m= lines that the endpoint expected to be bundled would have to be rejected, and since balanced is the default, this would mean that two non-bundled video streams would by default be rejected.

The alternative would simply be to say that if the remote side doesn't support BUNDLE, you just try to carry on without bundling, for current and future m= lines. This is a bit different than rtcp-mux, in that it is global, and therefore you can't get into weird cases depending on who is offering. But then the symmetry with the rtcp-mux policy is lost.